core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -511,17 +519,14 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
     val cromwellSubmission = for {
       (wdl, workflowRecs, wfInputsBatch, wfOpts, wfLabels, wfCollection, dosUris, petUserInfo, methodConfig) <-
         workflowBatchFuture
-      dosServiceAccounts <- resolveDrsUriServiceAccounts(dosUris, petUserInfo)
+      dosSignedUrls <- resolveDrsSignedUrls(dosUris, petUserInfo)
       // For Jade, HCA, anyone who doesn't use Bond, we won't get an SA back and the following line is a no-op
       // We still call DRSHub for those because we can verify the user has permission on the DRS object as
       // early as possible, rather than letting the workflow(s) launch and fail
       // AEN 2020-09-08 [WA-325]
       _ <-
-        if (dosServiceAccounts.isEmpty) Future.successful(false)
-        else
-          googleServicesDAO.addPolicyBindings(GoogleProjectId(wfOpts.google_project),
-                                              Map(requesterPaysRole -> dosServiceAccounts.map("serviceAccount:" + _))
-          )
+        if (dosSignedUrls.isEmpty) Future.successful(false)
+        else Future.successful(true) // TODO Is this correct? Or just don't check that it's empty at all?
```

Comment:
I'm a bit unclear on expected behavior - if *some* providers return a signed url, are we good to go?  Or will any lack of access fail the workflow so we should only consider success if we have a result for every provider?

Reply:
And failure here includes a null, not just an error thrown?  I guess I'm uncertain since we don't use the result at all, so it's unclear if it matters if it's null, only if a 403 is thrown.

Reply:
> I assumed we'd want to fail the submission if any of the DRS resolutions fail.

Correct. When DRS URIs are provided as inputs to workflows, the workflow infrastructure automatically resolves the DRS URI to a signed URL, and uses the signed URL to localize/download the referenced blob to the local file system of the VM running the job. If any DRS URI fails to localize/download, the job fails. If a DRS URI fails to resolve to a signed URL, it is better to fail during submission than to fail downstream after the VM has started.

> this would also allow us to name the failing DRS host in a log message or in the error

Great! Identifying the failing host/provider in the error message would be extremely helpful to users and Frontline Support.